### PR TITLE
fft modified and new files to compare synchrotron frequency and horizontal tune variations

### PR DESCRIPTION
**Pull request by mauro-di-liddo**
Here in main and other codes I have changed fftshift()->fft() in order to obtain spectra not centered, trying to fix so the main peak at f_sample/2;
Function 'compare_fsy' used to compare different values of synchrotron frequency to be evaluated on the same plot: results to be overlapped to the same curve, so the contribute to the motion can be considered quite independent from the 'fs' value;
Function 'compare_qh' used to compare different values of horizontal tune to be evaluated on two different plots: the first one is related to the transversal unbunched beam motion and the second to the bunched transversal motion. Furthermore we can also change 'friv' value to evaluate more different betatron frequencies.

### CODE REVIEW

File: compareFsy.m [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+end
+
+x1=[t,x11,x12,x13];
+X11=fft(x11);
+X12=fft(x12);
+X13=fft(x13);
+X1=[f,X11,X12,X13];
+plotTimesFreqfig(x1,X1);
+title('Curves with different synchrotron frequency values');
+legend('Longi Bunched w/ fs(1)','Longi Bunched w/ fs(2)','Longi Bunched w/ fs(3)');
```
Comment (amereghe) [inline compareFsy.m]: please add a new line char at the end of the file

File: compareQh.m [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+Z1=[f,Z11,Z12,Z13]; %fft trans bunched matrix
+
+%plot
+plotTimesFreqfig(z0,Z0);
+title('Transversal unbunched curves with different horizontal tune values');
+legend('qh=1.656','qh=1.666','qh=1.676')
+hold on;
+plotTimesFreqfig(z1,Z1);
+title('Transversal bunched curves with different horizontal tune values');
+legend('qh=1.656','qh=1.666','qh=1.676')
```
Comment (amereghe) [inline compareQh.m]: please add a new line char at the end of the file

**Review by amereghe — CHANGES_REQUESTED** [review]
Hi @mauro-di-liddo ,

the PR is in general fine with me. I would honestly try to avoid committing main codes which are similar to each other; on the contrary, I would maybe have a function generating the signals based on some input parameters; then, in the main, we should see only the definition of the parameters, the call to the function as many times as the number of signals that we want, afterwards, the FFT and finally the plotting.

**Comment by mauro-di-liddo** [comment]
I would try to write two different functions rather then scripts and call them into the main or in a different one for comparing plots.

**Comment by mauro-di-liddo** [comment]
new files added, no need to correct old scripts

**Pull request merged**